src/sbvr-api/permissions.ts

```diff
@@ -1666,21 +1668,18 @@ export const addPermissions = async (
 		}
 	}
 
-	// This bypasses in the root cases, needed for fetching guest permissions to work, it can almost certainly be done better though
-	let permissions = req.user?.permissions ?? [];
-	permissions = permissions.concat(req.apiKey?.permissions ?? []);
+	let permissionsLookup;
```

Comment:
NIT: I think this atm is `any` .

Reply:
Yeah, I'm not sure why that doesn't error on the typings though (since we have `strict: true` which defaults `noImplicitAny: true`)? Either way I've explicitly typed it now